src/libxrpl/protocol/STInteger.cpp

```diff
@@ -171,6 +175,26 @@ template <>
 std::string
 STUInt32::getText() const
 {
+    if (getFName() == sfPermissionValue)
+    {
+        auto const permissionValue =
+            static_cast<GranularPermissionType>(value_);
+        auto const granular =
+            Permission::getInstance().getGranularName(permissionValue);
+
+        if (granular)
+        {
+            return *granular;
```

Comment:
According to [codecov](https://app.codecov.io/gh/XRPLF/rippled/commit/1ea0ef3fc2ab0727f49346235d3717a06d969046), this line is not tested